docker/env.example

```diff
@@ -0,0 +1,43 @@
+
+### LLM ###
+
+OPENAI_API_KEY=
+ANTHROPIC_API_KEY=
+
+###  Auth providers ###
+
+GITHUB_CLIENT_ID=
+GITHUB_CLIENT_SECRET=
+
+GOOGLE_CLIENT_ID=""
+GOOGLE_CLIENT_SECRET=
+
+LINKEDIN_CLIENT_ID=
+LINKEDIN_CLIENT_SECRET=""
+
+MICROSOFT_CLIENT_ID=
+MICROSOFT_CLIENT_SECRET=""
```

Comment:
Let's make these consistent and either use empty string `""` or blank (not sure which is better)